modules/drivers/robosense/decoder/decoder_16.hpp

```diff
@@ -0,0 +1,355 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#pragma once
+#include <memory>
```

Comment:
Please run `scripts/apollo_format.sh modules/drivers/robosense` first to format cpp/bazel files, and 
run `scripts/mdfmt.sh  modules/drivers/robosense` for to format README.md  written in markdown.